src/configure/configure.ts

```diff
@@ -154,6 +157,23 @@ class Orchestrator {
         }
     }
 
+    private async getGithubPatToken(): Promise<void> {
+        const redirectHelper = new RedirectLinkHelper();
+        await redirectHelper.loadAll();
+        if (this.inputs.sourceRepository.repositoryProvider === "github" || redirectHelper.repoAnalysisServiceFramework === ServiceFramework.Moda) {
```

Comment:
Should we throw an error in else case as it will be undesired behaviour?

Reply:
It is a valid flow - Azure Repo + Azure Pipeline. In that scenario we don't call Repo Analysis